### ISSUE TITLE
View Only mode kicks off logged in user.

### ISSUE THREAD

**Issue by FickoDeLux** [mRemoteNG/mRemoteNG#1727]
## Expected Behavior
Connecting to running session (User logged in) should not kick off the user in "View Only mode"

Implementation should use shadowing:

On host:
`
HKLM\SOFTWARE\Policies\Microsoft\Windows NT\Terminal Services\Shadow set to 4
`

Client:
`
mstsc.exe /v:<server> /shadow:<ID> /noConsentPrompt
`

`
query session /server:<server>
`
output could be used for parsing for ID.

## Current Behavior
Openning "View Only" session kicks off logged in user

OS:
Windows 10

**Comment by Benchwork** [comment]
are you doing this through external tools> powershell? or how are you trying to achieve this ? 
I have been trying to do something similar, 
I know that if you are kicking the user off, mstsc.exe is not connecting to the client with the shadow function correctly. 

note: different shadow settings in host.
1. No remote control allowed: Disallows an administrator to use remote control or view a remote user session.
2. Full Control with user's permission: Allows the administrator to interact with the session, with the user's consent.
3. Full Control without user's permission: Allows the administrator to interact with the session, without the user's consent.
4. View Session with user's permission: Allows the administrator to watch the session of a remote user with the user's consent.
5. View Session without user's permission: Allows the administrator to watch the session of a remote user without the user's consent.

**Comment by FickoDeLux** [comment]
I was talking about the current implementation.
Since it kicks off the user it has no real usage.

I am just proposing a different way of implementation.
Shadowing is great, only draw back the window can not be embedded.

**Comment by Benchwork** [comment]
so when I am looking through the config settings I don't see a setting for view only mode for RDP connections. 

but I do see this under the connect (with options)
![image](https://user-images.githubusercontent.com/7304823/84442534-cea8ad00-ac0b-11ea-90da-a89fcfc2c151.png)
is this where your talking about?

**Comment by FickoDeLux** [comment]
Yes.On Jun 11, 2020 23:50, Peter Christenson <notifications@github.com> wrote:
so when I am looking through the config settings I don't see a setting for view only mode for RDP connections.
but I do see this under the connect (with options)

is this where your talking about?

**Comment by robertpopa22** [comment]
Triage refresh (P2): this issue has been inactive for a long time. Please retest on the latest build and share reproducible steps, expected vs actual behavior, and logs/screenshots. If there is no confirmation/update, maintainers can close it as stale or not reproducible.

**Comment by robertpopa22** [comment]
Triage refresh (P4 version-label debt): this issue is tagged with legacy version labels. Please retest on the latest build (fork v1.79.x or upstream dev), then share reproducible steps and expected vs actual behavior. If still valid, keep open and relabel to current triage scope (for example Need 2 check). If fixed, close with resolution.

Current legacy version labels: 1.77.2
